docs/morevp.md

```diff
@@ -391,10 +391,11 @@ The following scenario demonstrates this attack:
 2. `TX1` is in-flight.
 3. Operator begins withholding blocks while `TX1` is still in-flight.
 4. Alice starts an exit referencing `TX1` and places `exit bond`.
+4. Alice piggybacks onto the exit and places `piggyback bond`.
 5. Mallory spends `UTXO1m` in `TX2`.
 6. In period 1 of the exit for `TX1`, Mallory challenges the canonicity of `TX1` by revealing `TX2`.
 7. No one is able to respond to the challenge in period 2, so `TX1` is determined to be non-canonical.
-8. After period 2, Mallory receives Alice's `exit bond`, no one exits any UTXOs.
+8. After period 2, Mallory receives Alice's `exit bond`, Alice receives `UTXO1a` and `piggyback bond`.
 
 Mallory has therefore caused Alice to lose `exit bond`, even though Alice was acting honestly.
```

Comment:
Can we add a comment that the Mallory is real looser here because she basically burned her `UTXO1m`... This will help to highlight that this attack comes at cost.

Reply:
Hm, you're right, with a few caveats:
1/ this is conditional on the fact, that none of the double-spent inputs can be piggybacked and exited. I think this piece of the protocol is not to depend on and there's a slight chance that letting exactly one of such piggybacks out will be useful elsewhere.
2/ Mallory could craft `TX1` so that her loss is miniscule - as long as this doesn't make Alice suspicious, e.g. she could split her input to `TX1` into two portions, e.g. 1 wei + 10 eth and double spend only the 1 wei

I'll make a note of this